releasenotes/notes/enable-enhanced-scope.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+releaseNotes:
+- |
+  **Updated** default value of the feature flag `ENABLE_AUTO_ENHANCED_RESOURCE_SCOPING` to true.
+upgradeNotes:
+- title: Default value of the feature flag `ENHANCED_RESOURCE_SCOPING` to true
+  content: |
+    `ENHANCED_RESOURCE_SCOPING` is enabled by default. This means that meshConfig.discoverySelectors
```

Comment:
Can you clarify a few points?

1. CustomResource meaning Istio resources?
2. limit configurations ....meaning "only process based on the scope of what is specified from meshConfig.discoverySelectors?

Reply:
I update the release notes to make it more clear